filclient.go

```diff
@@ -1049,25 +1069,59 @@ func (fc *FilClient) RetrieveContentWithProgressCallback(
 					}
 				case retrievalmarket.DealStatusRejected:
 					finish(fmt.Errorf("deal rejected: %s", resType.Message))
-				case retrievalmarket.DealStatusFundsNeededUnseal:
-					finish(fmt.Errorf("received unexpected payment request for unsealing data"))
+				case retrievalmarket.DealStatusFundsNeededUnseal, retrievalmarket.DealStatusUnsealing:
+					finish(fmt.Errorf("data is sealed"))
 				case retrievalmarket.DealStatusCancelled:
 					finish(fmt.Errorf("deal cancelled: %s", resType.Message))
 				case retrievalmarket.DealStatusErrored:
 					finish(fmt.Errorf("deal errored: %s", resType.Message))
 				case retrievalmarket.DealStatusCompleted:
 					finish(nil)
-				default:
-					log.Debugf("in-progress voucher response status: %v", retrievalmarket.DealStatuses[resType.Status])
 				}
-			default:
-				log.Debugf("in-progress voucher result type: %v", resType)
 			}
-		case datatransfer.DataReceived:
+		case datatransfer.PauseInitiator:
```

Comment:
you can put multiple case values on a single line, like:
```
case datatransfer.PauseInitiator, datatransfer.ResumeInitiator:
```

Might make things a little cleaner? not sure

Reply:
I considered that, it didn't really seem any more readable due to the amount of cases though, and this way it'll be easier to add actions to cases without having to split up the comma list and move things around and add new cases n stuff

your call :-)